### ISSUE TITLE
NPE when inspecting container

### ISSUE THREAD

**Issue by adrianbk** [bmuschko/gradle-docker-plugin#170]
Version: 2.6.5

An NPE occurs when inspecting a running container with the following task
```
task inspectContainer(type: DockerInspectContainer, group: 'docker') {
    containerId = 'my-container'
}
```

The image of the container in question has 3 volumes as follows:
```
VOLUME  ["/etc/dir1", "/etc/dir2", "/etc/dir3"]
```

The `DockerInspectContainer` task does produce some output as follows but then fails:
```
Inspecting container with ID 'my-container'.
Image ID    : sha256:xxxxxxxxxxxxxxxxxxxxxxxxxx
Name        : /my-container
Links       : []
:myProject:inspectContainer FAILED
```

Stacktrace:

```
Caused by: java.lang.NullPointerException
	at com.github.dockerjava.api.command.InspectContainerResponse.getVolumes(InspectContainerResponse.java:129)
	at com.bmuschko.gradle.docker.response.container.InspectContainerResponseHandler.handle(InspectContainerResponseHandler.groovy:15)
	at com.bmuschko.gradle.docker.response.container.InspectContainerResponseHandler.handle(InspectContainerResponseHandler.groovy)
	at com.bmuschko.gradle.docker.response.ResponseHandler$handle.call(Unknown Source)
	at com.bmuschko.gradle.docker.tasks.container.DockerInspectContainer.runRemoteCommand(DockerInspectContainer.groovy:28)
	at com.bmuschko.gradle.docker.tasks.AbstractDockerRemoteApiTask$_start_closure1.doCall(AbstractDockerRemoteApiTask.groovy:48)
	at com.bmuschko.gradle.docker.utils.DockerThreadContextClassLoader.withClasspath(DockerThreadContextClassLoader.groovy:40)
	at com.bmuschko.gradle.docker.utils.ThreadContextClassLoader$withClasspath.call(Unknown Source)
	at com.bmuschko.gradle.docker.tasks.AbstractDockerRemoteApiTask.start(AbstractDockerRemoteApiTask.groovy:47)
```

**Comment by cdancy** [comment]
@adrianbk might be an issue that was solved with 2.6.6. Can you try with that? I know you're currently having issues elsewhere but I'd like to see if 2.6.6 solves your problem. I believe the issue you note stems from the docker-java library we use as our base and it throwing an NPE when trying to get list of volumes.

**Comment by adrianbk** [comment]
@cdancy It works with `2.6.6`. Thanks.
